Document/0x05b-Basic-Security_Testing.md

```diff
@@ -766,7 +775,7 @@ Discuss with your project team the possibility of providing a debug build for th
 
 If the app implements certificate pinning, C.509 certificates provided by an interception proxy will be declined and the app will refuse to make any requests through the proxy. To perform an efficient white box test, use a debug build with deactivated certificate pinning.
```

Comment:
Can you change the C.509 into X.509?

Reply:
> Can you change the C.509 into X.509?

Added in commit [d3c97ba](https://github.com/OWASP/owasp-mstg/commit/d3c97ba55406669811637cb088b639812581a39f)